style.css

```diff
@@ -28,7 +28,15 @@ header {
   padding: 20px;
 }
 
-a {
+@media (min-width: 800px) {
+  .header {
+    display: grid;
+    grid-template-columns: 25% 75%;
+    padding: 20px 10%;
+  }
+}
```

Comment:
- [ ] There are over 40 appearances of 
``` html
@media (min-width: 800px) {
      selector {
         property: value;
         property: value;
         property: value; 
     }
}
```

which makes your code difficult to read. Kindly bring them all together instead of repeating yourself like so:

``` html
@media (min-width: `x`px) {
      selector-1 {
         property: value;
         property: value;
         property: value; 
      }
      selector-2 {
         property: value;
         property: value;
         property: value; 
      }

      selector-3 {
         property: value;
         property: value;
         property: value; 
      }

}
```
- [ ] You currently have 800px as the min-width for the desktop screen-size which is too small. Kindly use something between (992px - 1100px)